From d1df15ecca61b294de34b4c5e3708a4344f75468 Mon Sep 17 00:00:00 2001
From: vipinc <vipinc@ami.com>
Date: Fri, 29 Dec 2023 01:39:35 +0530
Subject: [PATCH 1/1] Add Restart always to unit service file

This patch update bmcweb.service unit service with
Restart=always configuration. This helps to confirm systemd
will make an attempt to restart the bmcweb.service incase
the service is unsuccessfully exited due to any case

Tested in INTEL Archercity:
- Verified bmcweb process is respawned even after killing the
process

Signed-off-by: vipinc <vipinc@ami.com>
---
 config/bmcweb.service.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/config/bmcweb.service.in b/config/bmcweb.service.in
index a07adbe6..d7a7b4c8 100644
--- a/config/bmcweb.service.in
+++ b/config/bmcweb.service.in
@@ -5,6 +5,7 @@ Wants=network.target
 After=network.target
 
 [Service]
+Restart=always
 ExecReload=kill -s HUP $MAINPID
 ExecStart=@MESON_INSTALL_PREFIX@/bin/bmcweb
 ExecStop=/bin/bash /usr/bin/delete-hi-user.sh
-- 
2.25.1

